src/components/ComponentSettingsModal.jsx

```diff
@@ -0,0 +1,467 @@
+                }
+                {/* Permalink */}
+                <FormField
+                  title="Permalink"
+                  id="permalink"
+                  value={permalink ? permalink : ''}
+                  errorMessage={errors.permalink}
+                  isRequired
+                  onFieldChange={this.changeHandler}
+                  disabled={!isPost}
+                  fixedMessage={category ? `/${category}/` : '/'}
+                  placeholder=' '
+                />
+                { type === "resource" && 
+                // Resource Type
+                <>
+                  <div className="d-flex row m-0">
+                    <div className="col-4 m-0 p-0">
+                      {/* Date */}
+                      <FormField
+                        title="Date (YYYY-MM-DD)"
+                        id="date"
+                        value={date}
+                        errorMessage={errors.date}
+                        isRequired
+                        onFieldChange={this.changeHandler}
+                      />
+                    </div>
+                    
+                    <div className="col-8">
+                      <p className={elementStyles.formLabel}>Resource Type</p>
+                      {/* Permalink or File URL */}
+                      <div className="d-flex">
+                        <label htmlFor="radio-post" className="flex-fill">
+                          <input
+                            type="radio"
+                            id="radio-post"
+                            name="resource-type"
+                            value="post"
+                            onClick={this.handlePermalinkFileUrlToggle}
+                            checked={isPost}
+                          />
+                          Post Content
+                        </label>
+                        <label htmlFor="radio-post" className="flex-fill">
+                          <input
+                            type="radio"
+                            id="radio-file"
+                            name="resource-type"
+                            value="file"
+                            onClick={this.handlePermalinkFileUrlToggle}
+                            checked={!isPost}
+                          />
+                          Downloadable File
+                        </label>
+                      </div>
+                    </div>
+                  </div>
+                  {/* File URL */}
+                  <FormFieldFile
+                    title="Select File"
+                    id="fileUrl"
+                    value={fileUrl?.split('/').pop()}
+                    errorMessage={errors.fileUrl}
+                    onFieldChange={this.changeHandler}
+                    inlineButtonText="Select File"
+                    disabled={isPost}
+                    onInlineButtonClick={() => this.setState({ canShowFilesModal: true })}
+                  />
+                </>}
```

Comment:
Likewise, this can be a separate component

Reply:
Resolved in https://github.com/isomerpages/isomercms-frontend/pull/184/commits/7b07257ca347731c531c7e6e9a4d5fbeeabd8064